library/src/main/java/com/novoda/downloadmanager/lib/jobscheduler/DownloadJob.java

```diff
@@ -0,0 +1,47 @@
+package com.novoda.downloadmanager.lib.jobscheduler;
+
+import android.content.Context;
+import android.content.Intent;
+import android.support.annotation.NonNull;
+
+import com.evernote.android.job.Job;
+import com.evernote.android.job.JobRequest;
+import com.novoda.downloadmanager.lib.DownloadService;
+import com.novoda.downloadmanager.lib.GlobalState;
+import com.novoda.downloadmanager.lib.logger.LLog;
+
+import java.util.concurrent.TimeUnit;
+
+public class DownloadJob extends Job {
```

Comment:
I would keep this in the `lib` package - are you expecting clients of the library to instantiate this? (i don't think so)